### PR TITLE
HOMS-200 Use ruby:slim for base docker image

### PR DESCRIPTION
**Pull request by pleaseduck**

### CODE REVIEW

**Comment by sugarfree1** [comment]
Change Dockerfile.oracle and Dockerfile.base.

**Comment by pleaseduck** [comment]
upd

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+  libxslt-dev \
+  make \
   nodejs \
-  postgresql-client
+  postgresql-client \
+  pkg-config \
+  ruby-dev
 
-RUN useradd --uid 2004 --home /opt/homs --shell /bin/bash --comment "HOMS" homs
+ENV NLS_LANG          AMERICAN_RUSSIA.AL32UTF8
```
Comment (sugarfree1) [inline Dockerfile]: Why not ```NLS_LANG=AMERICAN_RUSSIA.AL32UTF8```?

File: Dockerfile.oracle [hunk 2]
```diff
@@ -62,5 +69,5 @@ RUN cp -r /opt/homs/config/* /tmp/config
 EXPOSE 3000
 
 USER homs
-
+WORKDIR /opt/homs
```
Comment (sugarfree1) [inline Dockerfile.oracle]: You added WORKDIR here to ```Dockerfile.oracle```, but didn't add to ```Dockerfile```.

**Comment by pleaseduck** [comment]
upd

**Comment by pleaseduck** [comment]
upd

**Comment by pleaseduck** [comment]
upd

**Pull request merged**